containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/mariadb-client-install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script is used to install a matching mariadb-client
+# in ddev-webserver
+# It should be called with the appropriate mariadb version as an argument
+# This script is intended to be run with root privileges (normally in a docker build phase)
+
+set -eu -o pipefail
+
+DDEV_DATABASE_FAMILY=${DDEV_DATABASE%:*}
+if [ "${DDEV_DATABASE_FAMILY}" != "mariadb" ]; then
+  echo "This script is to be used only with a project using mariadb" && exit 1
+fi
+MARIADB_VERSION=${DDEV_DATABASE#*:}
+# Add all required MariaDB versions here
```

Comment:
It's hard to understand this comment. Are we supposed to add more required MariaDB versions? I guess maybe it means "All mariaDB versions that require the compat packages?"